PyramidLights/PyramidLightsMain.cs

```diff
@@ -22,59 +29,63 @@ static void Main(string[] args)
-                    {
-                        //tweet question
-                        TextInfo myTI = new CultureInfo("en-US", false).TextInfo;
+                    if (question.endDateTime < DateTime.Now)
+                        break;
 
-                        if (!question.tweetTooLong())
+                    if (DateTime.Now > question.startDateTime && DateTime.Now <= question.endDateTime)
+                        if (question.questionText != null)
                         {
-                            long questionTweetID = twitterHelper.sendTweet(question.tweetString + " " + DateTime.Now.ToString());
+                            //tweet question
+                            TextInfo myTI = new CultureInfo("en-US", false).TextInfo;
 
-                            //var twitterTask = Task.Run(async () => await cTwitterHelper.ProcessTweet(questionTweetID, question));
-                            //listen for responses 
-                            twitterHelper.listenAndProcess(questionTweetID, question);
+                            if (!question.tweetTooLong())
+                            {
+                                //long questionTweetID = twitterHelper.sendTweet(question.tweetString + " " + DateTime.Now.ToString());
+                                long questionTweetID = twitterHelper.sendTweet(question.tweetString);
 
-                        }
+                                //var twitterTask = Task.Run(async () => await cTwitterHelper.ProcessTweet(questionTweetID, question));
+                                //listen for responses 
+                                twitterHelper.listenAndProcess(questionTweetID, question);
+                            }
 
-                    }
-                    //dtResetDay = DateTime.Now;
+                        }
 
                     //if (dtStartToday.AddDays(1) > dtResetDay && dtStartToday.Day != dtResetDay.Day)
                     //{
                     //    break;
                     //}
-
-                    if (question.endDateTime < DateTime.Now)
-                        break;
                 }
+                System.Diagnostics.Debug.WriteLine("End of while loop in PyramidLightsMain. Question Expired. Getting new question.");
 
             }
         }
         catch (Exception Ex)
         {
-            System.Diagnostics.EventLog.WriteEntry("PyramidLightsError", Ex.ToString(), System.Diagnostics.EventLogEntryType.Error);
-
-            try
-            {
-                PyramidLights.Classes.cMail.SendEmail(configHelper, Ex.Message);
-            }
-            catch(Exception ignoreEx)
-            {
-                System.Diagnostics.EventLog.WriteEntry("PyramidLightsError-Mail", ignoreEx.ToString(), System.Diagnostics.EventLogEntryType.Error);
-            }
+            System.Diagnostics.EventLog.WriteEntry(EventLogSource, Ex.ToString(), System.Diagnostics.EventLogEntryType.Error);
+           
+            //try
```

Comment:
In the future we can just delete things instead of commenting them out as it will be in the git history if needed.